llama_stack/cli/stack/run.py

```diff
@@ -47,12 +47,17 @@ def _add_arguments(self):
             default=os.environ.get("CONDA_DEFAULT_ENV"),
             help="Name of the image to run. Defaults to the current environment",
         )
-        self.parser.add_argument(
+        hostargs = self.parser.add_mutually_exclusive_group()
+        hostargs.add_argument(
             "--disable-ipv6",
             action="store_true",
             help="Disable IPv6 support",
             default=False,
         )
+        hostargs.add_argument(
+            "--host",
+            help="Interface on which to listen",
+        )
```

Comment:
Can we avoid adding more args and just keep the config in the config object?

Reply:
Yes, also please see https://github.com/meta-llama/llama-stack/pull/1105 :)

Reply:
I will update the PR. Thanks for the response!